odl/contrib/fom/util.py

```diff
@@ -393,3 +394,36 @@ def haarpsi_weight_map(img1, img2, axis):
     np.abs(img2_lvl3, out=img2_lvl3)
 
     return np.maximum(img1_lvl3, img2_lvl3)
+
+
+def radial_sum(image, binning_factor=1):
+    """Sum image values over concentric annuli.
```

Comment:
From the name of the funciton, I'd probably expect the cumulative sum instead (e.g. sum over circles)